tests/unit/datastore_test.py

```diff
@@ -808,6 +903,15 @@ def mock_read_no_pools(path):
     result.children = []
     return result
 
+def mock_read_no_bgppeers(path):
+    """
+    EtcdClient mock side effect for read with no IPv4 BGP Peers
+    """
+    result = Mock(spec=EtcdResult)
+    assert path == BGP_PEERS_PATH
+    result.children = []
```

Comment:
`result.children = iter([])`

This will help catch bugs where datastore.py doesn't respect the python-etcd API and tries to perform list operations on what is really an iterator.